core/domain/user_services_test.py

```diff
@@ -47,6 +47,25 @@ def test_set_and_get_username(self):
         user_services.set_username(user_id, username)
         self.assertEquals(username, user_services.get_username(user_id))
 
+    def test_get_username_for_system_user(self):
+        self.assertEqual(
+            feconf.SYSTEM_COMMITTER_ID,
+            user_services.get_username(feconf.SYSTEM_COMMITTER_ID))
+        self.assertEqual(
+            feconf.MIGRATION_BOT_USERNAME,
+            user_services.get_username(feconf.MIGRATION_BOT_USER_ID))
+
+    def test_get_usernames(self):
```

Comment:
Add a few more tests, try to break it. E.g. empty list, or trying to find users who don't exist.

Reply:
Added these two cases.